src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -136,14 +149,26 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
             LOGGER.debug("Notification through EMAIL");
             String subject = getSubscriptionField("emailSubject", subscriptionJson);
             try {
-                sendMail.sendMail(notificationMeta, String.valueOf((mapNotificationMessage.get("")).get(0)), subject);
+                emailSender.sendMail(notificationMeta, String.valueOf((mapNotificationMessage.get("")).get(0)), subject);
```

Comment:
emailSender sends mail? if correct it should be emailSender.sendEmail or mailSender.sendMail. pick one :)